abcix/example/kvstore/kvstore.go

```diff
@@ -112,19 +117,22 @@ func (app *Application) CreateBlock(
 
 		newState, gasUsed, err := executeTx(app.state, tx, true)
```

Comment:
tx should be executed in previous state:

```golang
tempState := app.state
for ... {
  newState = executeTx(tempState, tx, true)
  ...
  tempState = newState
}
```

then app hash will be `tempState.Size`